docs/usage-guide/book.toml

```diff
@@ -0,0 +1,5 @@
+[book]
+authors = []
+language = "en"
+multilingual = false
+src = "src"
```

Comment:
If we get to choose the directory where all the docs go, we should choose a better name than "src". Maybe "topics"? We probably want something that makes sense if you're just looking in the source for the documentation too, since that was the previous story.